tests/settings.py

```diff
@@ -186,6 +186,14 @@
 
 EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
 
+# Make sure the search index is updated immediately during tests
+TASKS = {
+    "default": {
+        "BACKEND": "django_tasks.backends.immediate.ImmediateBackend",
+        "ENQUEUE_ON_COMMIT": False,
```

Comment:
This took me way too long to figure out. I need to reach for the [release notes](https://docs.wagtail.org/en/stable/releases/6.4.html#background-tasks-run-at-end-of-current-transaction) more often 🙃